src/main/java/seedu/address/logic/commands/CreateCommand.java

```diff
@@ -0,0 +1,94 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_COMPANIES;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.company.Company;
+import seedu.address.model.poc.Poc;
+
+/**
+ * Creates a new poc to be added to the specified company.
+ */
+public class CreateCommand extends Command {
```

Comment:
A well written class that doesn't violate any of the code quality standards 👍

Reply:
Thank you Elgin